experiment/template/eVOLVER.py

```diff
@@ -97,8 +102,11 @@ def on_broadcast(self, data):
         for param in temp_cal['params']:
             self.save_data(data['data'].get(param, []), elapsed_time,
                         VIALS, param + '_raw')
-        # run custom functions
-        self.custom_functions(data, VIALS, elapsed_time)
```

Comment:
Do we really want nothing to happen until this countdown is finished? I feel like the countdown should be used within the custom functions - this way it can be up to the logic within the function to decide how it wants to handle the restart.